src/nsls2api/services/facility_service.py

```diff
@@ -136,10 +134,22 @@ async def data_admin_group(facility_name: str) -> Optional[str]:
     """
     facility = await Facility.find_one(Facility.facility_id == facility_name)
 
-    if facility is None:
-        return None
+    return facility.data_admin_group if facility else None
 
-    return facility.data_admin_group
+
+async def get_data_admins(facility_name: str) -> list[str]:
+    """
+    Retrieves the data admins for a given facility name.
+
+    Args:
+        facility_name (str): The facility name. e.g. "nsls2, lbms, cfn, etc."
+
+    Returns:
+        list[str]: A list of data admins for the specified facility.
+    """
+    facility = await Facility.find_one(Facility.facility_id == facility_name)
```

Comment:
Maybe this function parameter should be called `facility_id` rather than `facility_name` to avoid confusion with `Facility.name`?

Reply:
I agree - I've noticed lots of inconsistencies that I've built up - I think a major renaming refactoring is on the cards! 🤯